android/app/src/main/java/com/bounswe2020group3/paperlayer/home/EventFragment.kt

```diff
@@ -94,13 +97,7 @@ class EventFragment : Fragment(), HomeContract.EventView {
         viewAdapter.notifyDataSetChanged() //notify to update recyclerview
         writeLogMessage("i", TAG,"Project Card List Updated! " + eventCardsList.size)
     }
-    fun test(){
-        addCard(EventCard("a","a","a","a","a","a"))
-        addCard(EventCard("b","b","b","b","b","b"))
-        addCard(EventCard("c","c","c","c","c","c"))
-        submitCardList()
 
-    }
     override fun getLayout(): View {
         TODO("Not yet implemented")
```

Comment:
Leaving a TODO item in production is  not a good practice but if you don't need it in near future, it is OK I guess